### PR TITLE
Adding posibility to resize table columns

### PR DESCRIPTION
**Pull request by hrigner**
It might be a bit of a "hidden feature", since it's only visible when hovering over between tables.. but I like it that way.

### CODE REVIEW

**Comment by sublibra** [comment]
Looks great!

File: src/components/model.jsx [hunk 1]
```diff
@@ -156,171 +155,208 @@ export default function Model({ app, appLayout }) {
...
 
-                const assocStyle = {
-                  backgroundColor: x.backgroundColor,
-                };
+        localStorage.setItem(key, ref.offsetWidth);
+      }
+    };
+
+    const key = `${app.id}/tables/${tableName}`;
+    const minTableWidth = 29 * 8;
```
Comment (peol) [inline src/components/model.jsx]: Magic numbers, maybe use constants?

File: src/components/model.jsx [hunk 2]
```diff
@@ -156,171 +155,208 @@ export default function Model({ app, appLayout }) {
...
+    const savedTableWidth = (isLocalStorage && localStorage.getItem(key));
+    const tableSize = savedTableWidth ? { width: `${savedTableWidth}px` } : 'auto';
+    const handleStyle = {
+      right: {
+        zIndex: 3,
+        width: '3.5em',
```
Comment (peol) [inline src/components/model.jsx]: Can this be based off of the `29 * 8` stuff above so it would scale if we tune the numbers?

Comment (hrigner) [reply src/components/model.jsx]: The 3.5 em is connected to the the column padding, and 29 * 8 is sth I totally made up myself 😂 
But I get your message, I'll think of sth!

**Pull request merged**